src/background/index.ts

```diff
@@ -48,7 +48,15 @@ if (WATCH) {
                 }
             }
         };
-        socket.onclose = () => setTimeout(listen, 1000);
+        socket.onclose = () => {
+            const ALARM_NAME = 'socket-close';
+            chrome.alarms.onAlarm.addListener((alarm) => {
```

Comment:
`setTimeout` will only call the code once. But with this, it will call every second and act as an `setInterval`

Reply:
This patch specifies only `delayInMinutes`, which means alarm will be triggered only once like `setTimeout`. To get a recurring event I would have to use `periodInMinutes`.

Reply:
Aha, didn't know the `alarms` API allowed that.